docs-kits/kits/Industry Core Kit/Software Development View/part_uniqueidpush.mdx

```diff
@@ -28,7 +28,9 @@ The solution is based on notification assets in the EDC (which is the same appro
 
 > :raised_hand: It is important to understand that the receiver creates EDC asset and policies, and thus, the sender of the Unique ID push notification must check during the EDC negotiation process if the conditions the receiver offers are acceptable for the sender.
 
-Currently there are two types of Unique ID Push Notifications available: "Connect to Parent" and "Connect to Child".
+Currently there are two types of Unique ID Push Notifications available: "Connect to Parent" and "Connect to Child":
+
+![Unique ID Push Notifications](../assets/unique_id_push_notifications.svg)
```

Comment:
In the picture, please change "must" to "can"
![Screenshot 2024-10-17 162306](https://github.com/user-attachments/assets/4a6718e2-7cd7-4300-91b1-bdb2e84fdeb8)

Reply:
Done! 😄